drivers/pds/lib/utils.go

```diff
@@ -630,7 +630,9 @@ func GetVersionsImage(dsVersion string, dsBuild string, dataServiceID string) (s
 	isVersionAvailable = false
 	isBuildAvailable = false
 	for i := 0; i < len(versions); i++ {
-		if (*versions[i].Enabled) && (*versions[i].Name == dsVersion) {
+		log.Debugf("version name %s and is enabled=%t", *versions[i].Name, *versions[i].Enabled)
+		if *versions[i].Name == dsVersion {
+			log.Debugf("DS Version %s is enabled in the control plane", dsVersion)
```

Comment:
here enable check is not happening . is it intentional?

Reply:
yes, we don't need that check as suggested by developers.